bench/utils/__init__.py

```diff
@@ -0,0 +1,85 @@
+import struct
+import sys
+import time
+import random
+
+_RANDOM = random.Random(0)
```

Comment:
I'm pretty nervous about using `0` as a seed here: older versions of Python have problems with the MT whereby seeds that have long sequences of NULL bytes initially produce random output that is not evenly distributed for quite a while. It'd be better to use a 128-bit random seed generated from urandom on your own machine and just hard-code that into the file with a comment that says why we did it.